### PR TITLE
[test] | [canary] Move latest image definitions to utils

### PR DESCRIPTION
**Pull request by arjkesh**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*
- Maintain dictionary of latest images in utils. This will be replaced by an autogenerated json file in the near future.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/test_utils/__init__.py [hunk 1]
```diff
@@ -324,6 +331,47 @@ def get_dlc_images():
...
+    tf2 = "2.2"
+    mx = "1.6"
+    pt = "1.5"
+
+    if framework == "tensorflow":
+        framework = "tensorflow2" if "tensorflow2" in os.getenv("CODEBUILD_BUILD_ID") else "tensorflow1"
+
+    images = {
+        "tensorflow1":
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-gpu-py3 "
```
Comment (nskool) [inline test/test_utils/__init__.py:346]: Nit: trailing spaces.

File: test/test_utils/__init__.py [hunk 2]
```diff
@@ -324,6 +331,47 @@ def get_dlc_images():
...
 
+def parse_canary_images(framework, region):
+    tf1 = "1.15"
+    tf1_long = "1.15.2"
+    tf2 = "2.2"
+    mx = "1.6"
+    pt = "1.5"
+
+    if framework == "tensorflow":
+        framework = "tensorflow2" if "tensorflow2" in os.getenv("CODEBUILD_BUILD_ID") else "tensorflow1"
```
Comment (nskool) [inline test/test_utils/__init__.py:342]: Can you paste sample value of CODEBUILD_BUILD_ID for record?

File: test/test_utils/__init__.py [hunk 3]
```diff
@@ -324,6 +331,47 @@ def get_dlc_images():
...
+
+    images = {
+        "tensorflow1":
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-gpu-py3 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-cpu-py3 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-cpu-py2 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-training:{tf1}-gpu-py2 "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-inference:{tf1}-gpu "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-inference:{tf1}-cpu "
+            f"763104351884.dkr.ecr.{region}.amazonaws.com/tensorflow-inference:{tf1_long}-gpu-py27-cu100-ubuntu18.04 "
```
Comment (nskool) [inline test/test_utils/__init__.py:352]: Are all these images supported in SM?

Comment (arjkesh) [reply test/test_utils/__init__.py:352]: they are not, but I don't think any TF py2 inference images are

File: test/test_utils/__init__.py (hunk 1 above)
Comment (arjkesh) [reply test/test_utils/__init__.py:346]: These are strings, space separated on purpose.

File: test/test_utils/__init__.py (hunk 2 above)
Comment (arjkesh) [reply test/test_utils/__init__.py:342]: See https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html

```
CODEBUILD_BUILD_ID: The CodeBuild ID of the build (for example, codebuild-demo-project:b1e6661e-e4f2-4156-9ab9-82a19EXAMPLE).
```

File: test/test_utils/__init__.py (hunk 3 above)
Comment (nskool) [reply test/test_utils/__init__.py:352]: It should be alright to remove non-SM images from this list.

**Pull request merged**